### PR TITLE
Link Implementation for Bridge Driver

### PR DESCRIPTION
**Pull request by mavenugo**

### CODE REVIEW

File: drivers/bridge/bridge.go [hunk 1]
```diff
@@ -568,11 +575,113 @@ func (d *driver) DeleteEndpoint(nid, eid types.UUID) error {
...
+	}
+
+	if endpoint.config != nil && endpoint.config.PortBindings != nil {
+		for _, p := range cc.parentEndpoints {
+			var parentEndpoint *bridgeEndpoint
+			parentEndpoint, err = network.getEndpoint(p)
+			if err != nil {
+				return err
+			}
+			if parentEndpoint == nil {
```
Comment (mrjana) [inline drivers/bridge/bridge.go]: Actually this is an error. When this list is provided during `Join` all of the endpoints (both parents and children) need to exist. So we should fail this case as well.

File: drivers/bridge/bridge.go [hunk 2]
```diff
@@ -568,11 +575,113 @@ func (d *driver) DeleteEndpoint(nid, eid types.UUID) error {
...
+		}
+	}
+
+	for _, c := range cc.childEndpoints {
+		var childEndpoint *bridgeEndpoint
+		childEndpoint, err = network.getEndpoint(c)
+		if err != nil {
+			return err
+		}
+		if childEndpoint == nil {
```
Comment (mrjana) [inline drivers/bridge/bridge.go]: Same here. We should fail this case as well.

File: drivers/bridge/bridge.go (hunk 1 above)
Comment (mavenugo) [inline drivers/bridge/bridge.go]: hmm okay... I wasn't too sure of that. Anyways, handled this as an error case as well.

**Comment by mrjana** [comment]
LGTM

**Pull request merged**